src/com/github/igotyou/FactoryMod/Factorys/NetherFactory.java

```diff
@@ -449,7 +445,7 @@ protected void recipeFinished() {
 		switch(mode) {
 		case REPAIR:
 			repairMaterials.addAll(netherFactoryProperties.getRepairMaterials());
-			repairMaterials = repairMaterials.getMultiple(netherManager.getScalingFactor(factoryLocation));
+			repairMaterials.getMultiple(((NetherFactoryManager) FactoryModPlugin.getPlugin().manager.getManager(NetherFactoryManager.class)).getScalingFactor(factoryLocation));
```

Comment:
Don't drop the assignment. getMultiple does not modify the repairMaterials object, but returns a new clone. This patch would break repair scaling.